packages/caver-transaction/src/transactionTypes/abstractTransaction.js

```diff
@@ -390,15 +390,7 @@ class AbstractTransaction {
      * await tx.fillTransaction()
      */
     async fillTransaction() {
-        const [chainId, gasPrice, nonce] = await Promise.all([
```

Comment:
Do we need to implement a fillTransaciton() function in each transaction? How about checking whether createTxObj is DynamicFeeTx or not?

Reply:
Or How about overloading this function in DynamicFeeTx?

Reply:
I don't understand what `How about checking whether createTxObj is DynamicFeeTx or not?` means.
And the reason why i removed this logic here is for less Promise wait.
To override function, i need to separate logic for (chainId and nonce) and gasPrice, because gasPrice is no more defined in `AbstractTransaction`.
I can change to implement some variables(chainId and nonce which are common variable for all tx types) fetching in AbstractTransaction class, and override in each tx types to fetch gasPrice or additional logic for DynamicFee tx.
But in my opinion, this implementing separate might be better.